.github/workflows/UnitTests.yml

```diff
@@ -110,6 +110,10 @@ jobs:
       run: |
         docker run -v /home/runner/actions-runner/_work/maxtext/maxtext:/app --rm --privileged maxtext_base_image bash -c \
         'bash end_to_end/test_generate_param_only_checkpoint.sh -r runner_$(date +%Y-%m-%d-%H-%M)-${RANDOM} -o gs://runner-maxtext-logs -d gs://maxtext-dataset -i 4'
+    - name: Test generate_param_only_checkpoint with int8 quantization
```

Comment:
There were some code paths that our test suites missed that broke with these changes. This should cover them

Reply:
Oh interesting, we haven't really considered generating quantized checkpoints -- we assumed quantization would be in the node after loading. How would we load a quantized checkpoint?

Reply:
This question of any special considerations around loading a quantized checkpoint is on my mind for the fp8 work especially. With AQT/int8 today, I believe we are just loading params as is and then doing a dummy inference to quantize the weights (and then saving those quantized weights). This test covers that code path (see https://github.com/google/maxtext/pull/554/files#diff-f521b06e43d6c7cab9e2f65a5eb4fa64cd129e63f1796154b31b42cf4481af68R99). 

It'll get more complex with FP8, where we'll have histories and scales that are variables of their own. I'll need to do some digging.